modules/manage/pages/manage-security/manage-users-and-roles.adoc

```diff
@@ -35,110 +35,178 @@ image::manage-security/usersAndGroupsTab.png[,170,align=left]
+
+To define a group, left-click on the *Add Group* tab, near the upper right of the *Security* screen:
+
+image::manage-security/addGroupTab2.png[,210,align=left]
+
+This brings up the *Add New Group* dialog:
+
+[#add-new-group-dialog]
+image::manage-security/addNewGroupDialog.png[,420,align=left]
+
+The fields are as follows:
+
+* *Group Name*.
+The name of the new Couchbase-Server group to be created.
+
+* *Description*.
+An optional description of the new Couchbase-Server group.
+
+* *Map to LDAP Group*.
+The name of an existing LDAP group to which the new Couchbase-Server group is to be mapped.
+For details, see xref:manage:manage-security/configure-ldap.adoc#map-ldap-groups-to-couchbase-server-roles[Map LDAP Groups to Couchbase Server Roles].
+
+* *Roles*. The roles to be associated with the new Couchbase-Server group.
+The display of roles is divided into sections.
+The first section, which appears at the top of the panel, is for *Administration & Global Roles*: these roles are either administrative, or involve access to cluster-wide features.
+Below the first group, the *All Buckets (&#42;)* group appears: roles in this section apply to all buckets defined on the cluster.
+Below *All Buckets (&#42;)*, additional sections are displayed, one for each individual bucket currently defined on the cluster.
++
+This arrangement allows the new group to be assigned roles that apply either to all buckets defined on the cluster, or to one or more particular buckets.
+Left-click on the corresponding right-pointing arrowhead to display a section's contents, scrolling down if necessary:
++
+[#all_buckets_checkboxes]
+image::manage-security/allBucketsCheckboxes.png[,280,align=left]
++
+To display roles at lower levels of the *All Buckets (&#42;)* hierarchy, again, left-click on the right-pointing arrowheads.
+To assign roles to the new group, simply check the appropriate checkboxes.
+(Note that some roles are considered to be _subsets_ of others.
+In such cases, manually checking one checkbox may trigger the automated checking of others — indicating that the corresponding roles are also assigned to the new group.)
+
+To create a `ClusterAdmin` group, each of whose members is granted the *Cluster Admin* role, enter the following:
+
+[#add-new-group-dialog-no-mapping]
+image::manage-security/addNewGroupDialogNoMapping.png[,420,align=left]
+
+As this indicates, no LDAP group mapping is required to define a group intended for the support only of users who are defined on Couchbase Server as _local_ or _external_.
+For an explanation of LDAP group mappings, see xref:manage:manage-security/configure-ldap.adoc#map-ldap-groups-to-couchbase-server-roles[Map LDAP Groups to Couchbase Server Roles].
+
+Save the group by left-clicking on the *Save* button.
+Note that the group will not be displayed by Couchbase Web Console until at least one user has been created and assigned to it.
```

Comment:
Doesn't seem to be correct. I think that's a UI bug. Groups should be visible even if there is no users created and groups are empty.

Reply:
Okay, I have removed this statement.